index.js

```diff
@@ -23,12 +23,15 @@ function fastifyInjector (injectorOpts = {}, fastify = require('fastify')()) {
         delete injectors[prop][name]
         return target[prop](name, injectDecorator)
       }
-      return target[prop](name, value)
+      target[prop](name, value)
+
+      // Return the fastify instance to support method chaining.
+      return this
     }
   }
 
   function wrapRegisterMethod (register) {
-    return function wrappedRegister (originalPlugin, opts) {
+    return function wrappedRegister (originalPlugin, opts, other) {
```

Comment:
Do we need to have `other` as an argument here?

Reply:
nope, leftover debug. Thanks!